lib/stdlib/src/io_lib_format.erl

```diff
@@ -649,7 +660,19 @@ insert_decimal(Place, S) ->
                     end;
                 true ->
                     if
-                        Place - L + 2 =< ExpCost ->
+                        %% All integers in the range [-2^53, 2^53] can
+                        %% be stored without lose of precision in an
```

Comment:
*loss* of precision (multiple occurrences)

Reply:
Thanks for seeing this!

Reply:
Fixed in 9443da9